app/containers/Applications/ApplyButtonContainer.tsx

```diff
@@ -53,11 +53,7 @@ export const ApplyButton: React.FunctionComponent<Props> = ({
   ]);
 
   if (!canOpenApplication) {
-    return (
-      <Button disabled variant="info">
-        The application window is closed
-      </Button>
-    );
+    return null;
```

Comment:
Returning a (not visible, empty div) in this case would prevent the last table column from collapsing when there are no buttons present in that column. (The column's width is determined by its contents):

```suggestion
    return (
      <div style={{min-width: '9.7em'}} />
    );
```